content/docs/concepts/security/index.md

```diff
@@ -535,6 +535,48 @@ runtime. When a request comes to the proxy, the authorization engine evaluates
 the request context against the current authorization policies, and returns the
 authorization result, `ALLOW` or `DENY`.
 
+### Authorization permissive mode
+
+Authorization permissive mode allows users to verify authorization policies
+before applying them in production environment.
+
+Authorization permissive mode could be set on both global authorization
+configuration and individual policies. When setting permissive mode on global
+authorization configuration, all policies will be in permissive mode regardless
+its own mode; otherwise If the global authorization configuration is set to
+`ENFORCED`, the enforcement mode set on individual policy takes effect.
```

Comment:
Maybe mention that "If `mode` is not specified, it defaults to `ENFORCED`"?

Reply:
added.